Header header
float32 range
float32 velocity
float32 bearing
uint8 quality
uint8 track_id
uint8 msg_counter
uint8 sensor_nr
uint8 target_format_type
uint8 object_type
